docs/design/coreclr/profiling/davbr-blog-archive/DoStackSnapshot - Callback CONTEXT Registers.md

```diff
@@ -5,9 +5,9 @@ In my initial [post](DoStackSnapshot - Exception Filters.md) about DoStackSnapsh
 
 The quick answer is that **nonvolatile (i.e., preserved), integer registers** should be valid.  You don't really need many registers to walk the stack anyway.  Obviously, you want a good stack pointer and instruction pointer.  And hey, a frame pointer is handy when you come across an EBP-based frame in x86 (RBP on x64).  These are all included in the set, of course.  Specifically by architecture, you can trust these fields in your context:
 
-x86: Edi, Esi, Ebx, Ebp, Esp, Eip  
-x64: Rdi, Rsi, Rbx, Rbp, Rsp, Rip, R12:R15  
+x86: Edi, Esi, Ebx, Ebp, Esp, Eip
+x64: Rdi, Rsi, Rbx, Rbp, Rsp, Rip, R12:R15
```

Comment:
This is again an **undesired** visible change. The two trailing spaces were treated as a `<br>`. After removing them, these appear in the same line.